File: Back_End/Models/Reviews/Review.cs
using Back_End.Models.Books;
using Back_End.Models.Users;

namespace Back_End.Models.Reviews
{
    public class Review : BaseEntity.BaseEntity
    {
        public string Title { get; set; }
        public string Content { get; set; }
        public int Rating { get; set; } // 1-5
        public Book Book { get; set; }
        public Guid BookId { get; set; }
        public User User { get; set; }
        public Guid UserId { get; set; }
    }
}

File: Front_End/src/app/data/interfaces/review.ts
import { User } from "./user";
import { User2 } from "./user2";

export interface Review {
  title: string;
  content: string;
  rating: number;
  bookId: string;
  userId: string;
  user: User2;
  dateCreated: Date;
  dateModified: Date;
}
